package.json

```diff
@@ -82,6 +82,7 @@
     "vue-recaptcha": "^3.0.0-alpha.6",
     "vue-router": "^4.1.6",
     "vue3-text-clamp": "^0.1.2",
+    "workbox-precaching": "^7.1.0",
```

Comment:
Is this related to the `workbox-precaching` error I have been seeing in local recently?

Reply:
I think so, I had to install it. It worked for me! I hope it helps, otherwise I can take it out :)